skymap_scanner/client/client.py

```diff
@@ -41,7 +63,9 @@ def _create_dir(val: str) -> Optional[Path]:
             "The directory with the 'startup.json' file to startup the client "
             "(has keys 'mq_basename', 'baseline_GCD_file', and 'GCDQp_packet')"
         ),
-        type=Path,
+        type=lambda x: wait_for_file(
+            x, cfg.env.SKYSCAN_CLIENT_WAIT_FOR_STARTUP_JSON_SEC, "startup.json"
+        ),
```

Comment:
What is the purpose of this wait?  I can't imagine this is useful in production, since the startup file needs to be transferred with the job.

Reply:
If we put the startup.json somewhere else, like `/data/user/`, then this is useful. It also means we can launch everything at once, so the clients can startup while the initial pixels are being generated (which can take a while on a full-size scan)

Reply:
yeah, I see your point now. Moving that logic upstream...